Jarvis-Platform/jarvis-studio
# fix(LogsComponent): Add dag_type field in query

[comment]: [![Status](https://img.shields.io/badge/Status-DEVELOPMENT-orange.svg)]()
[comment]: [![Status](https://img.shields.io/badge/Status-HOLD-inactive.svg)]()

[![Status](https://img.shields.io/badge/Status-READY-brightgreen.svg)]()


## Description
Add `dag_type` field in logs query.


## Todos
- [x] Add field from props


## Deploy Notes
Review & merge


## Steps to Test or Reproduce
```sh
git checkout master
git pull
git checkout hotfix/dag-type-on-logs
```

1. Go to a `tables-to-tables` task listing
2. Press logs btn => Logs should be decoded and displayed


## Impacted Areas in Application
- `src/components/data-workflows/common/item/logs/LogsComponent.vue`
